granulate_utils/linux/cgroups/cgroup.py

```diff
@@ -88,13 +198,46 @@ def get_cgroup_mount(controller: str, resolve_host_root_links: bool = True) -> O
     v2_path = find_v2_hierarchy(resolve_host_root_links)
     if v1_paths and controller in v1_paths:
```

Comment:
```suggestion
    if controller in v1_paths:
```

Reply:
Btw is this check correct? For a controller I suppose we allow `cpu` and the underlying v1 controller might be `cpu,cpuacct`; what will happen in this case?

Reply:
Yeah it looks correct to me:
`iter_mountinfo` will return `cpu` and `cpuacct` separated and then `find_v1_hierarchies` adds them as a separate key to the returned dictionary. However the path it returns will look like `/sys/fs/cgroup/cpu,cpuacct` but that's the way it works today.